apps/web/components/ActionCategoryCard/ActionCategoryCard.tsx

```diff
@@ -141,22 +141,23 @@ const Component = forwardRef<HTMLElement, ActionCategoryCardProps>(
             display="flex"
             justifyContent={['flexStart', 'flexEnd']}
             flexDirection="row"
+            style={{ cursor: cta.disabled ? 'not-allowed' : '' }}
```

Comment:
It's probably harmless to set the cursor to an empty string but maybe it's safer to set it to undefined so it behaves the default way

Reply:
Agree, fixed